moped-editor/src/views/projects/mapHelpers.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
+
+export const MAPBOX_TOKEN = `pk.eyJ1Ijoiam9obmNsYXJ5IiwiYSI6ImNraWV4dHR0ZjAwNnYyd3FwYjFoNHduaDcifQ.--3vRm2KHq1gh5K_L0pqtA`;
```

Comment:
This should be part of the environment variables I think

Reply:
Cool, it is restricted to staging, production, and dev for now but I think that y'all are right that we should hide it away. I'll create an issue to delete this token, create a fresh one, and update where this comes from. 👍